packages/fxa-payments-server/src/routes/Subscriptions/ActionButton.test.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { screen, render, cleanup, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import ActionButton, { ActionButtonProps } from './ActionButton';
+import { CUSTOMER } from '../../lib/mock-data';
+import { PickPartial } from '../../lib/types';
+import { defaultConfig } from '../../lib/config';
+
+const { apiUrl } = defaultConfig().paypal;
+
+describe('routes/Subscriptions/ActionButton', () => {
+  type SubjectProps = PickPartial<
+    ActionButtonProps,
+    'customer' | 'onRevealUpdateClick' | 'revealFixPaymentModal'
+  >;
+  const Subject = ({
+    customer = CUSTOMER,
+    onRevealUpdateClick = jest.fn(),
+    revealFixPaymentModal = jest.fn(),
+  }: SubjectProps) => {
+    return (
+      <ActionButton
+        {...{
+          customer,
+          onRevealUpdateClick,
+          revealFixPaymentModal,
+        }}
+      />
+    );
+  };
+
+  afterEach(() => {
+    return cleanup();
+  });
+
+  it('renders revealonclick button when payment_provider is "stripe"', async () => {
+    const onRevealUpdateClick = jest.fn();
+    render(<Subject onRevealUpdateClick={onRevealUpdateClick} />);
+    const revealButton = screen.getByTestId('reveal-payment-update-button');
+    expect(revealButton).toBeInTheDocument();
+    fireEvent.click(revealButton);
+    expect(onRevealUpdateClick).toBeCalled();
+  });
+
+  it('renders paypalActionButton when paypal is payment provider and paypal_payment_error is not set', async () => {
+    render(
+      <Subject
+        customer={{
+          ...CUSTOMER,
+          payment_provider: 'paypal',
+        }}
+      />
+    );
+    const revealButton = screen.getByTestId('change-payment-update-button');
+    expect(revealButton).toBeInTheDocument();
+    expect(revealButton.getAttribute('href')).toEqual(
+      `${apiUrl}/myaccount/autopay/connect/${CUSTOMER.billing_agreement_id}`
```

Comment:
> 
> In that case we should file a Bugzilla bug for setting this value in prod. 

We need a value for that regardless since the default is the sandbox...?  Maybe I understand why this patch is the reason for adding that env var for prod.

Reply:
I will add to the Deploy Doc

Reply:
I opened this PR to update the environment variable in prod: https://github.com/mozilla-services/cloudops-infra/pull/2993 .